jck/jck_target_folder_check/jck_target_folder_check.sh

```diff
@@ -96,9 +96,9 @@ crossCheckTestFoldersIn() {
 		existingTargets=$( cat "${listOfExistingTargets}" )
 		if [[ ! " $existingTargets " =~ $tokenToSearch ]] ; then
 			echo "New test folder detected: $line | Please add target: jck-$tokenToSearch"
-			command='<command>$(JCK_CMD_TEMPLATE) -test-args=$(Q)tests='${tokens[1]}/${tokens[2]}',jckRoot=$(JCK_ROOT),jckversion=$(JCK_VERSION),testsuite='$testTypeToUpper'$(Q); \'
-			status='$(TEST_STATUS)</command>'
-			echo "<test>"
+            command='<command>$(JCK_CMD_TEMPLATE) tests='${tokens[1]}/${tokens[2]}' -DjckRoot=$(JCK_ROOT) -Djckversion=$(JCK_VERSION) -Dtestsuite='$testTypeToUpper'; \
+            $(TEST_STATUS)</command>'
+            echo "<test>"
```

Comment:
Since `JCK_CMD_TEMPLATE` has been modified to use `JavatestUtil` in another file, other parameters seem to be standard, I think we can keep this command as is? And the indentation should be adjusted too.